docker/README.md

```diff
@@ -83,7 +83,9 @@ As a result of this command, there is a context broker listening on port 1026 on
 A few points to consider:
 
 * The name `orion1` can be anything and doesn't have to be related to the name given to the docker image in the previous section.
-* `--link mongodb:mongodb` assumes there is a docker container running a MongoDB image in your system, whose name is `mongodb`. In case you need one type `sudo docker run --name mongodb -d mongo:2.6`.
-* In `-p 1026:1026` the first value represents the port to listen in on localhost. If you wanted to run a second context broker on your machine you should change this value to something else, for example `-p 1027:1026`.
+* `--link mongodb:mongodb` assumes there is a docker container running a MongoDB image in your system,
+which name is `mongodb`. In case you need one type `sudo docker run --name mongodb -d mongo:2.6`.
```

Comment:
You are partially right, you can use 'whose', but according to 

http://dictionary.cambridge.org/grammar/british-grammar/relative-pronouns

whose is for people and animals usually; sometimes for things in formal situations
which is for animals and things

thus which is correct but as you were the original author I'm keeping 'whose' as I want the change to land as soon as possible :)